src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Certificates/StartCertificateHandlerTests/StartCertificateHandlerTests.cs

```diff
@@ -320,5 +332,118 @@ public void SetUp()
 
             _mockCertificateRepository.Verify(s => s.Update(existingCertificate, request.Username, null, true, null), Times.Once);
         }
+
+        [Test, RecursiveMoqAutoData]
+        public async Task WhenHandlingStartCertificateRequest_AndNamesAreInLowerCase_ThenRunProperCaseAlgorithm(StartCertificateRequest request,
```

Comment:
These tests check whether the ProperCase algorithm is run by relying on what it does; if it actually returned the string which was passed in then I think they would still pass; one way you could solve that would be to refactor the helper class into a ProperCaseService which is injected into StartCertificateRequest, as IProperCaseService you can then check that this was actually called by verifying that a Mock was called when the names where all one case and not called when they were mixed case without relying on what the algorithm does.

Reply:
I don't think I understand. If it returned the string that has been passed in, the test would fail because the assertions check that the output values are equivalent to the input value being put through the algorithm.